server/lib/payments.js

```diff
@@ -192,6 +192,7 @@ const processPayment = (donation) => {
               hostFeeInTxnCurrency: parseInt(balanceTransaction.amount * hostFeePercent / 100, 10),
               platformFeeInTxnCurrency: fees.applicationFee,
               paymentProcessorFeeInTxnCurrency: fees.stripeFee,
+              description: donation.title,
```

Comment:
This field is meant to be deprecated - it's better to keep info in donation or expense and pull it from there. That way, transaction remains a purely ledger entry and all info about the original intent is separate.